examples/correlation_alignment_example.py

```diff
@@ -0,0 +1,84 @@
+"""Align data with significant frequency drift
+==============================================
+
+Takes a 2D data set and applies proper phasing corrections followed by 
+aligning the data through a correlation routine.
+"""
+from pyspecdata import *
+from pyspecProcScripts import *
+from pylab import *
+import sympy as s
+from collections import OrderedDict
+from numpy.random import normal,seed
+seed(2021)
+rcParams['image.aspect'] = 'auto' #needed for sphinx gallery
+
+#{{{generate fake data
+fl = figlist_var()
+#this generates fake clean data with a T2 of 0.2s
+# amplitude of 23 (arbitrary), offset of 300 Hz,
+# FWHM 10 Hz
+f_range = (-400,400)
+t2, td, vd, ph1, ph2 = s.symbols('t2 td vd ph1 ph2')
+echo_time = 1e-3
+data = fake_data(
+        23*(1 - 2*s.exp(-vd / 0.2)) * s.exp(+1j*2*s.pi*100*(t2) - abs(t2)*50*s.pi),
+        OrderedDict([
+            ('vd', nddata(r_[0:1:40j],'vd')),
```

Comment:
in the loop that I talk about above, this would also change (you would have a power, rather than vd axis)